.github/workflows/build.yml

```diff
@@ -347,10 +347,12 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Setup Minikube-Kubernetes
-        uses: manusa/actions-setup-minikube@v2.3.0
+        uses: manusa/actions-setup-minikube@v2.7.1
         with:
-          minikube version: v1.15.0
-          kubernetes version: 1.18.0
+          minikube version: 'v1.26.1'
+          kubernetes version: 'v1.25.0'
```

Comment:
Along with this should we update the k8s dependecies in our code. Currently its at k8s 1.20. The controller runtime also would have to be updated to a matching version for the k8s version.

Reply:
Updated the libraries.